src/components/3-organisms/automaton/AutomatonProperties.tsx

```diff
@@ -40,9 +42,9 @@ const AutomatonProperties = ({
                     propertyName,
                     propertyType
                 )(
-                    // convert an array to a string content with ',' delimiter
+                    // convert an array to a string content with VALUE_DELIMITER
                     _.isArray(propertyValue)
-                        ? _.join(propertyValue, ', ')
```

Comment:
why do you remove the space ?

Reply:
that's the problem to add the ids ok I see

Reply:
It is better with no space then trim(), it can tolerate two cases: with or without space..

Reply:
Anyway, this is a quickfix to make griddyna work at moment, in the future we will replace this autocomplete component by another in the study-app, cf; https://github.com/gridsuite/gridstudy-app/blob/main/src/components/utils/checkbox-autocomplete/checkbox-autocomplete.tsx